sdlib/d/gc/bitmap.d

```diff
@@ -170,6 +181,22 @@ public:
 		}
 	}
 
+	void setBitValueAtomic(bool V)(uint index) shared {
```

Comment:
You probably want to have setBitAtomic and clearBitAtomic that forward to this so the API is the same. Also, test.

Reply:
You don't test the atomicity of write, but at least you can test it does the right thing.